staticcheck.conf

```diff
@@ -0,0 +1,10 @@
+checks = ["all", "-ST1000", "-ST1003", "-ST1016"]
```

Comment:
Is it possible to locate this file somewhere other than the root folder in order to today and organize the project folder structure?

Reply:
We can hide the file by adding an `.` before file name if we don't want to show on root folder.

Reply:
That's nice, yeah. This frustrating bit about this approach is the marketing aspect of the layer5io/meshery repo. When people view that GitHub page, the hidden files still (appropriately) show in that view. I wonder if there's another "tools" folder we might consolidate this and similar files under. 

Not the end of the world. But, just a consideration as we go to create this file (before it gets semi-cemented in-place).

Reply:
How about putting all such files in `config folder`? They may support relative imports I suppose.

Reply:
@leecalcote Done reorganised the configuration files into tools folder!